web_ui/server.py

```
@@ -1867,66 +2077,59 @@ def login_page():
-                    auth_mod.create_user(username, password)
-                except ValueError as exc:
-                    return render_template("login.html", error=str(exc), show_2fa=_totp_enabled())
-                except Exception:
-                    # Non-fatal persistence error; continue without persisting.
-                    pass
+                return render_template("login.html", error="Credentials did not match", show_2fa=_totp_enabled())
+            try:
+                auth_mod.create_user(username, password)
+            except ValueError as exc:
+                return render_template("login.html", error=str(exc), show_2fa=_totp_enabled())
+            except Exception:  # noqa: S110
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)